### DISCUSSION TITLE
Cask wine-stable infected with malware

### DISCUSSION THREAD

**Discussion by mthurston-zl** [Homebrew/discussions#5116] [Casks] [answered]
### Output of `brew config`

```shell
irrelevant for discussion
```

### Output of `brew doctor`

```shell
irrelevant for discussion
```

### Description of issue

Installing wine over cask wine-stable (which installs wine 9.0 from https://github.com/Gcenx/macOS_Wine_builds/releases/)  contains malware:
/Applications/Wine Stable.app/Contents/Resources/wine/lib/wine/i386-windows/concrt140.dll - Gen:Variant.Tedy.519834
/Applications/Wine Stable.app/Contents/Resources/wine/lib/wine/i386-windows/mmdevapi.dll - Gen:Variant.Tedy.513477
/Applications/Wine Stable.app/Contents/Resources/wine/lib/wine/x86_64-windows/magnification.dll - Gen:Variant.Bulz.727055

### Relevant casks

wine-stable

**Comment by SMillerDev** [comment]
What makes you think that?

**Comment by Gcenx** [comment, marked as answer]
False positives; 
- [CrossOver 19 and antivirus programs](https://www.codeweavers.com/support/forums/general/?t=27;msg=222870)
- [Windows Defender detects Occamy.c trojan in steam proton 5.0 folder](https://github.com/ValveSoftware/Proton/issues/3593)

Simply most anti-virus/malware software are lazy and just flag windows binaries that seem to be replacements for the Microsoft shipped ones (anything they don’t have a checksum for)

This also occurs when wine is self built.
